ajax/visibility.php

```diff
@@ -82,7 +87,12 @@
                     'prefix'   => $_POST['prefix'],
                 ],
             ];
-
+            if (isset($_POST['entity']) && $_POST['entity'] >= 0) {
+                $params['entity'] = $_POST['entity'];
+                $params['toupdate']['moreparams']['entity'] = $_POST['entity'];
+                $params['entity_sons'] = $_POST['is_recursive'] ?? false;
+                $params['toupdate']['moreparams']['entity_sons'] = $_POST['is_recursive'] ?? false;
+            }
```

Comment:
This change does not seems related to current PR, is it a fix?

Reply:
I think it's for the next PR (Targeted followup / task / solution Templates) - I drop it & i check for the next.